backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1079,3 +1079,88 @@ databaseChangeLog:
         - addNotNullConstraint:
             tableName: device_type
             columnName: swab_type
+  - changeSet:
+      id: event-corrections-feature
+      author: tnielsen2@cdc.gov
+      comment: Add columns to existing tables to handle the concept of corrections to test_events.
+      changes:
+        - sql:
+            remarks: Create the enumerations for corrections_status in test_event table.
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.TEST_CORRECTION_STATUS as ENUM('ORIGINAL', 'CORRECTED', 'REMOVED');
+            rollback:
+              sql: |
+                DROP TYPE ${database.defaultSchemaName}.TEST_CORRECTION_STATUS;
+        - addColumn:
+            tableName: test_order
+            columns:
+              - column:
+                  name: correction_status
+                  type: ${database.defaultSchemaName}.test_correction_status
+                  remarks: Enumeration of type of correction. This column is used for non-delete corrections.
+                  defaultValue: 'ORIGINAL'
+                  value: 'ORIGINAL'
+                  constraints:
+                    nullable: false
+              - column:
+                  name: reason_for_correction
+                  type: text
+                  remarks: User supplied comment when a correction happens.
+        - addColumn:
+            tableName: test_event
+            columns:
+              - column:
+                  name: patient_answers_data
+                  type: jsonb
+                  remarks: The patient's demographics and address at the time the test was administered.
+              - column:
+                  name: date_tested_backdate
+                  type: DATETIME
+                  remarks: Timestamp to store a user correction to when the test was ordered. Copied from TestOrder. Null if not backdated.
+              - column:
+                  name: test_order_id
```

Comment:
Foreign key constraint should be added here, not after the fact—and after the fact we should make it non-nullable.